### PR TITLE
replaces shares with share capital

### PR DESCRIPTION
**Pull request by rudolfix**
ticket stuff moved to ticket:
#3450

### CODE REVIEW

File: packages/web/app/modules/investor-portfolio/selectors.ts [hunk 1]
```diff
@@ -174,7 +173,7 @@ export const selectIsEligibleToPreEto = (state: IAppState, etoId: string) => {
 };
 
 export const selectShouldShowWhitelistDiscount = (state: IAppState, eto: TEtoSpecsData) => {
-  const isPreEto = selectEtoOnChainStateById(state, eto.etoId);
+  const isPreEto = selectEtoOnChainStateById(state, eto.etoId) === EETOStateOnChain.Whitelist;
```
Comment (Moejoe90) [inline packages/web/app/modules/investor-portfolio/selectors.ts]: was this changed or a bug?

Comment (rudolfix) [reply packages/web/app/modules/investor-portfolio/selectors.ts]: a bug, we always shown discounted tickets ;/

**Comment by rudolfix** [comment]
SMALL TEXTING CHANGES TO THR FORM

in here:
image
change % of voting power -> % of voting quorum

in new field `SHAREHOLDERS VOTING QUORUM`
add text `% of voting power` -> same layout as above

why?
now you can specify required quorum for voting (how many votes out of all possible need to be cast in order for voting to be valid)
and majority rule - % of votest cast with yes in order for voting to pass

File: packages/web/app/components/eto/public-view/EtoInvestmentTermsWidget.tsx [hunk 2]
```diff
@@ -128,31 +130,54 @@ const EtoInvestmentTermsWidgetLayout: React.FunctionComponent<TExternalProps & T
...
+                label={<FormattedMessage id="eto.public-view.token-terms.authorized-capital" />}
+                value={
+                  <>
+                    <FormatNumber
+                      value={etoData.authorizedCapital}
+                      outputFormat={ENumberOutputFormat.INTEGER}
+                      inputFormat={ENumberInputFormat.FLOAT}
+                      defaultValue={<ToBeAnnounced />}
+                    />
+                    {` ${etoData.company.shareCapitalCurrencyCode}`}
```
Comment (rudolfix) [inline packages/web/app/components/eto/public-view/EtoInvestmentTermsWidget.tsx]: I know that it's a quick fix but currency should be part of the component (`FormatNumber`)

File: packages/web/app/components/eto/registration/pages/EtoVotingRights/EtoVotingRights.tsx [hunk 3]
```diff
@@ -148,6 +148,21 @@ const EtoVotingRightsComponent: React.FunctionComponent<
               />
             </div>
 
+            <div className="form-group">
+              <FormFieldLabel name="shareholdersVotingQuorum">
```
Comment (rudolfix) [inline packages/web/app/components/eto/registration/pages/EtoVotingRights/EtoVotingRights.tsx]: you can also fix dividend rights, it's one line and your other PR has conflicts

File: packages/web/app/lib/api/eto/EtoApi.interfaces.unsafe.ts [hunk 4]
```diff
@@ -441,8 +496,9 @@ export type TBookbuildingStatsType = {
 export type TEtoSpecsData = TEtoTermsType &
   TEtoEquityTokenInfoType &
   TEtoVotingRightsType &
-  TEtoInvestmentTermsType &
-  IAdditionalEtoType;
+  TEtoInvestmentTermsType & {
```
Comment (rudolfix) [inline packages/web/app/lib/api/eto/EtoApi.interfaces.unsafe.ts]: good!

**Pull request merged**